pyitau/pages.py

```diff
@@ -209,3 +238,50 @@ def full_invoice_op(self):
             self._text,
             flags=re.DOTALL,
         ).group(1)
+
+
+class PixPage(SoupPage):
+    @property
+    def pix_statements_op(self):
+        return re.search(
+            r"self.consultarLancamentos = function\("
+            r"periodo, pagina, filtro, ordenacao\){[\n\r\t\s]+"
+            r"self.erroApiLancamento = false;[\n\r\t\s]+"
+            r"\$.Ajax\({[\n\r\t\s]+"
+            r'dataType : "json"[\n\r\t\s,]+'
```

Comment:
**Suggestion:** Avoid using regular expressions for parsing HTML. Instead, use a HTML parsing library like BeautifulSoup.
```suggestion
    @property
    def pix_statements_op(self):
        soup = BeautifulSoup(self._text, 'html.parser')
        return soup.find('pix', attrs={'data-op': True})['data-op']
```